src/services/SQService.ts

```diff
@@ -0,0 +1,83 @@
+import SQS, {GetQueueUrlResult, MessageBodyAttributeMap, ReceiveMessageResult, SendMessageResult} from "aws-sdk/clients/sqs";
+import {Service} from "../models/injector/ServiceDecorator";
+import {Configuration} from "../utils/Configuration";
+import {PromiseResult} from "aws-sdk/lib/request";
+import {AWSError, config as AWSConfig} from "aws-sdk";
+/* tslint:disable */
+const AWSXRay = require("aws-xray-sdk");
+/* tslint:enable */
+
+/**
+ * Service class for interfacing with the Simple Queue Service
+ */
+@Service()
```

Comment:
Just wanted to understand how we are using decorator in this class. Have you tested the code without it? Is it something necessary for SQS?

Reply:
From what I understood the annotation is used to instantiate the SQS service class. In the tests it is injected manually. I saw that most of our lambdas are using dependency injection to take care of instantiating service classes.